src/Fixer/Whitespace/NoTrailingWhitespaceFixer.php

```diff
@@ -70,15 +70,6 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
```

Comment:
Yes, they should behave as the PHPDoc describe :)
The comment I ref. to is the one inside the method (line 75)
```
 // should be run after NoEmptyPhpdocFixer, [...] and NoUselessElseFixer.
```

that info is only relevant to this fixer